modules/tests/shared/src/test/scala/data/SemispaceCacheTest.scala

```diff
@@ -9,58 +9,4 @@ import org.scalacheck.Gen
 import org.scalacheck.Prop._
 
 class SemispaceCacheTest extends ScalaCheckSuite {
```

Comment:
Why remove?

Reply:
I don't have to but it's completely empty.

Reply:
Hm, where did those tests go then?